lib/animation.js

```diff
@@ -211,12 +211,9 @@ Animation.prototype.speed = function(speed) {
  * play
  *
  * Start a segment
- *
- * @param  {Object} opts Options: reverse, loop, progress
- *
  */
 
-Animation.prototype.play = function(opts) {
+Animation.prototype.play = function() {
```

Comment:
I think it was my intent to allow the user tomodify the currently running animation (duration, speed stuff like that). That never happened, and I can't think of a specific use case for it so I'm cool with this change.